### ISSUE TITLE
cd to env variable doesn't work

### ISSUE THREAD

**Issue by aresler** [ranger/ranger#2143]
Ranger 1.9.3

#### Steps to reproduce
1. Create an environment variable with path, e.g. `export dl="/home/user/Downloads"
2. In Ranger, try to `cd $dl`

#### Expected

Navigate to `/home/user/Downloads`

#### Actual

Nothing happens

**Comment by aresler** [comment]
If this is not supported, please change to feature request.

**Comment by toonn** [comment]
Not currently possible unless you go through `eval`.

**Comment by toonn** [comment]
Here's how: `:eval import os; cd = self.fm.get_command('cd'); cd(':cd ' + os.getenv('dl')).execute()`
Not exactly a pretty sight, I admit. But you could conceivably define a custom command for this.
Usually though you'd just make a keybind and be done with it, [look at the series `g` keybinds for example](TargetSelectorNoTargetsInProject) or you bookmark the directory.

**Comment by chu4ng** [comment]
I pushed pull request for this feature, but I use bookmarks for all of my often using dirs.
May be OP has all his dirs list in .shrc file and want use it for ranger. For this case may be need use script for compile bookmarks($HOME/.local/share/ranger/bookmarks) file from shell env variables.

**Comment by aresler** [comment]
@chu4ng 

>May be OP has all his dirs list in .shrc file and want use it for ranger

Exactly, I find it quite useful to create env variable for frequently-used directories, e.g. $projects, $documents, e.t.c.

>For this case may be need use script for compile bookmarks($HOME/.local/share/ranger/bookmarks) file from shell env variables.

Interesting, I will try, thanks.

**Comment by chu4ng** [comment]
@aresler 
Yesterday I tried this script for compile "g" bookmarks for sh and for ranger. It looks like a solution to the problem, may be?
https://github.com/LukeSmithxyz/voidrice/blob/master/.local/bin/shortcuts

**Comment by aresler** [comment]
@chu4ng Thanks for suggestion, I haven't tried the script yet, but I don't think "g" shortcuts is the perfect solution because, AFAIK, you can only assign one letter per shortcut, and keyboard is limited. 

For now I workaround this by aliasing "ranger" to "r" in zsh, and using it instead of cd, like `r $dir` for quick navigation. This requires exiting ranger before doing this though.

**Comment by chu4ng** [comment]
@aresler 
No, it's not. You can can only assign one letter per shortcut with \` and `m` for bookmarks. But with `g` you can set long sequences of characters. Example, I use `ghP` for cd to home/screenshots/pic directory.
`map ghP cd ~/Screenshots/pic` in somefile. And I use `source somefile` for include this one.
